utils/datautils.js

```diff
@@ -49,20 +60,19 @@ function updateClaim(claimId, status) {
 
 function createClaim(user, issueUrl, pullUrl, bounty, status) {
 
-    const claim = {
-        action: 'create',
-        user, issueUrl, pullUrl, bounty, status
-    };
-    fs.writeFile(__dirname + '/../audit/' + new Date().toISOString() + '.json', JSON.stringify(claim), () => {});
```

Comment:
@championswimmer  sorry, it must have been removed during a conflict. Restored now